testing_and_setup/compass/ocean/global_ocean/EC15to60/init/define_base_mesh.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import jigsaw_to_MPAS.mesh_definition_tools as mdt
+from jigsaw_to_MPAS.coastal_tools import signed_distance_from_geojson, \
+    compute_cell_width
+from geometric_features import read_feature_collection
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+
+def cellWidthVsLatLon():
+    """
+    Create cell width array for this mesh on a regular latitude-longitude grid.
+    Returns
+    -------
+       cellWidth : numpy.ndarray
+            m x n array, entries are desired cell width in km
+       lat : numpy.ndarray
+            latitude, vector of length m, with entries between -90 and 90,
+            degrees
+       lon : numpy.ndarray
+            longitude, vector of length n, with entries between -180 and 180,
+            degrees
+    """
+    dlon = 0.1
+    dlat = dlon
+    nlon = int(360./dlon) + 1
+    nlat = int(180./dlat) + 1
+    lon = np.linspace(-180., 180., nlon)
+    lat = np.linspace(-90., 90., nlat)
+
+    cellWidthSouth = mdt.EC_CellWidthVsLat(lat, cellWidthEq=30.,
+                                           cellWidthMidLat=60.,
+                                           cellWidthPole=35.,
+                                           latPosEq=7.5, latWidthEq=3.0)
```

Comment:
@vanroekel, the edge of the 30-km region around the equator is at +/- 7.5 degrees (`latPosEq`).  This could be increased to 10.0.  I would probably also increase `latWidthEq` to 4.0.

Reply:
@mark-petersen had suggested that the counter current might not need high resolution, which is why we had halved the high-res region from 30 degrees total to 15 degrees total.  It sounds like @vanroekel thinks it needs high res.

Reply:
I updated this as stated above.